src/rules/block-no-empty/__tests__/index.js

```diff
@@ -48,3 +48,17 @@ testRule(rule, {
     column: 18,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [undefined],
+  skipBasicChecks: true,
+  syntax: "sugarss",
+
+  reject: [{
+    code: ".a",
+    message: messages.rejected,
+    line: 1,
+    column: 3,
```

Comment:
Isn't this a problem, because there is no 3rd column? Won't text editors complain? If so, we should make it column 2.